support/rename.sh

```diff
@@ -7,18 +7,32 @@ appRoot=`dirname $0`/..
 
 [ -z "${newAppName}" ] && echo 'Missing required parameter newAppName' && exit 1
 
-grep -rI 'PepperoniAppTemplate' --exclude='rename.sh' $appRoot/* | tr ':' ' ' | awk '{print $1}' | uniq | xargs -I{} sed -i.bak "s/PepperoniAppTemplate/${newAppName}/g" {}
-grep -rI 'pepperoniapptemplate' --exclude='rename.sh' $appRoot/* | tr ':' ' ' | awk '{print $1}' | uniq | xargs -I{} sed -i.bak "s/pepperoniapptemplate/${newLowerCaseName}/g" {}
-find . -name '*.bak' -exec rm {} \;
+# gather all modification targets
```

Comment:
I guess that's not `jest` related and could've been done in a separate PR ;) I like adding the renamed files directly to git :) However, after running the new rename script I got the error in the simulator that the app hasn't been registered.